package.json

```diff
@@ -38,6 +38,7 @@
     "eslint": "^4.16.0",
     "eslint-config-developit": "^1.1.1",
     "karmatic": "^1.0.6",
-    "microbundle": "^0.4.3"
+    "microbundle": "^0.4.3",
+    "webpack": "^4.29.6"
```

Comment:
Why is Webpack included in the dependencies?

Reply:
Webpack is a dev-dependency of karmatic. I don't personally have webpack installed globally, but I suppose that could have been a better solution. 
```

skelly@AASKI617 MINGW64 ~/Projects/js/greenlet (master)
$ npm run test

> greenlet@1.0.1 test C:\Users\skelly\Projects\js\greenlet
> eslint *.js && npm run -s prepare && karmatic

Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
Build output to dist:
        311 B: greenlet.js
        311 B: greenlet.m.js
        386 B: greenlet.umd.js
module.js:545
    throw err;
    ^

Error: Cannot find module 'webpack'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\skelly\Projects\js\greenlet\node_modules\karmatic\dist\configure.js:133:30)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\skelly\Projects\js\greenlet\node_modules\karmatic\dist\index.js:4:33)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! greenlet@1.0.1 test: `eslint *.js && npm run -s prepare && karmatic`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the greenlet@1.0.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\skelly\AppData\Roaming\npm-cache\_logs\2019-03-08T21_31_46_234Z-debug.log

```

Reply:
In that case I think you suggestion of adding `webpack` as dev dependency is correct :+1: